java/libphonenumber/src/com/google/i18n/phonenumbers/Phonemetadata.java

```diff
@@ -0,0 +1,1119 @@
+/*
+ * Copyright (C) 2010 The Libphonenumber Authors
```

Comment:
I guess we can say it's the same copyright year, since we are resurrecting an old file. Maybe say in the commit which file it is based off.